resourcemgr/resourcemgr.c

```diff
@@ -245,7 +246,7 @@ typedef struct {
                                     //  flushed.
     UINT64 connectionId;            // Used to identify which connection owns the object,
                                     // sequence, or session.
-    void *nextEntry; // Next entry in the list; 0 to terminate list.                                
+    RESOURCE_MANAGER_ENTRY_PTR nextEntry; // Next entry in the list; 0 to terminate list.                             
 } RESOURCE_MANAGER_ENTRY;
 
 RESOURCE_MANAGER_ENTRY *entryList = 0;
```

Comment:
On line 215 above you go through the trouble of creating a type for a pointer to a RESOURCE_MANAGER_ENTRY structure. Great that you've replaced the void* in the structure with this but you've got ~20 uses of RESOURCE_MANAGER_ENTRY* in resourcemgr.c. My take on this is that we should pick one approach (RESOURCE_MANAGER_ENTRY* or RESOURCE_MANAGER_ENTRY_PTR) and use it consistently. Mixing these arbitrarily hurts readability.